docs/getting_started/installation/cpu.x86.inc.md

```diff
@@ -164,21 +164,84 @@ uv pip install dist/*.whl
 [https://gallery.ecr.aws/q9t5s3a7/vllm-cpu-release-repo](https://gallery.ecr.aws/q9t5s3a7/vllm-cpu-release-repo)
 
 !!! warning
-    If deploying the pre-built images on machines without `avx512f`, `avx512_bf16`, or `avx512_vnni` support, an `Illegal instruction` error may be raised. It is recommended to build images for these machines with the appropriate build arguments (e.g., `--build-arg VLLM_CPU_DISABLE_AVX512=true`, `--build-arg VLLM_CPU_AVX512BF16=false`, or `--build-arg VLLM_CPU_AVX512VNNI=false`) to disable unsupported features. Please note that without `avx512f`, AVX2 will be used and this version is not recommended because it only has basic feature support.
+    If deploying the pre-built images on machines without `avx512f`, `avx512_bf16`, or `avx512_vnni` support, an `Illegal instruction` error may be raised. See the build-image-from-source section below for build arguments to match your target CPU capabilities.
 
 # --8<-- [end:pre-built-images]
 # --8<-- [start:build-image-from-source]
 
+## Building for your target CPU
```

Comment:
Perhaps we can simplify this section via: 
- Just show a build command with all build arguments
- note by default ISAs will be auto detected
- note all arguments like ```VLLM_CPU_{ISA}``` are used for cross-compilation, set ```VLLM_CPU_{ISA}=true``` will build vLLM with ```ISA```, set ```VLLM_CPU_{ISA}=false``` will depends on the auto-detection
- provide some examples for different scenarios